scripts/deployment/deploy.sh

```diff
@@ -38,7 +17,32 @@ if [ -z ${COMMIT_ID} ]; then
     export COMMIT_ID="latest"
 fi
 
+if [ -z ${TRAVIS_BRANCH} ]; then
+    echo "Please set the TRAVIS_BRANCH first."
+fi
+
+env=${TRAVIS_BRANCH}
+
+if [[ ${env} == "production" ]]; then
+    HOSTNAME="evalai.cloudcv.org"
+elif [[ ${env} == "staging" ]]; then
+    HOSTNAME="evalai-staging.cloudcv.org"
+else
+    echo "Skipping deployment since commit not on staging or production branch."
+    exit 0
+fi
+
 case $opt in
+        auto_deploy)
+            ssh ubuntu@${HOSTNAME} -i scripts/deployment/evalai.pem -o StrictHostKeyChecking=no \
+            "cd ~/Projects/evalai && \
+            export AWS_ACCOUNT_ID=${AWS_ACCOUNT_ID} && \
+            export COMMIT_ID=${COMMIT_ID} && \
+            eval $(aws ecr get-login --no-include-email) && \
```

Comment:
Let's use `aws_login;` here.

Reply:
Yeah, already tried that but sadly cannot use a function call inside since this is a string that is passed in after ssh to the machine.